State of the wpi 2 (#347)
ZeusWPI/zeus.ugent.be: content/blog/18-19/state-of-the-wpi.md

---
author: Robbe Van Herck
title: "State of the WPI"
created_at: 3-10-2019
description: Een tl;dr van het afgelopen jaar
---

Beste Zeusers en Zeusinnen,

> Nu het volgende academiejaar alweer van start gegaan is, is het hoog tijd om eens terug te blikken naar het afgelopen jaar. Wat is er allemaal gebeurd in en rond onze geliefde kelder?

Reviewer: ```suggestion
Nu het nieuwe academiejaar alweer van start gegaan is, is het hoog tijd om eens terugblik te werpen op het afgelopen jaar. Wat is er allemaal gebeurd in en rond onze geliefde kelder?
```

## Events

Zeus heeft dit jaar verre van stil gezeten op het vlak van events. Bijna elke week had zijn eigen event. Hoog tijd dus om ze eens op te lijsten!

### Esoterische kennismakings-avond (27 september)

> Wat is er leuker dan het academiejaar te openen met een avond vol programmeertalen die niemand begrijpt (of wil begrijpen)? Hoewel het vooral gemikt was op de eerstejaars die mogelijk nog niet konden programmeren, was het ook voor ervaren computerfluisteraars eens leuk om domme programma's te schrijven in domme talen. Er was zelfs zo veel volk dat we mensen op de gang hebben moeten zetten omdat de kelder vol was!

Reviewer 2: ```suggestion
Wat is er leuker dan het academiejaar te openen met een avond vol programmeertalen die niemand begrijpt (of wil begrijpen)? Hoewel het event vooral gemikt was op de eerstejaars die mogelijk nog niet konden programmeren, was het ook voor ervaren computerfluisteraars eens leuk om domme programma's te schrijven in domme talen. Er was zelfs zo veel volk dat we mensen op de gang hebben moeten zetten omdat de kelder vol was!
```

<%= figure 'https://zeus.ugent.be/zeuswpi/maze.gif', 'Een programma dat twee getallen optelt in de taal maze.' %>

### Spelletjesavonden (3 oktober en 20 maart)

> Omdat Zeus meer doet dan computeren organiseerden we ook dit jaar twee spelletjesavonden. Fans van zowel bekende spelen als Concept als fans van meer obscure spelen waarbij men de regels zelf moet uitzoeken konden zich volledig uitleven.

Reviewer 2: ```suggestion
Omdat Zeus meer doet dan computeren, organiseerden we ook dit jaar twee spelletjesavonden. Fans van zowel bekende spelen als Concept, tot fans van meer obscure spelen waarbij men de regels zelf moet uitzoeken, konden zich volledig uitleven.
```

PS: Wist je dat een van deze spelletjesavonden een vermelding gekregen heeft in Schamper?

### De Linux Install Party (11 oktober)

> Net als elk jaar dwingen *ik bedoel* begeleiden we weer een hoop mensen in de wondere wereld van Linux. Dit jaar zonder al te grote problemen (lees: niet *iedereen* was zijn/haar installatie kwijt) en niets dat niet opgelost kan worden door het gewoon eens opnieuw te proberen, en nog eens, ... Het was zelfs zo'n groot succes dat we al aan het plannen waren voor een Linux Reinstall Party.

Reviewer 2: ```suggestion
Net als elk jaar dwingen *ik bedoel* begeleiden we weer een hoop mensen in de wondere wereld van Linux. Dit jaar ging dat zonder al te grote problemen (lees: niet *iedereen* was zijn/haar installatie kwijt). Er is niets dat niet opgelost kan worden door het gewoon eens opnieuw te proberen, en nog eens, ... Het was zelfs zo'n groot succes dat we al aan het plannen waren voor een Linux Reinstall Party.
```

> Net als elk jaar dwingen *ik bedoel* begeleiden we weer een hoop mensen in de wondere wereld van Linux. Dit jaar zonder al te grote problemen (lees: niet *iedereen* was zijn/haar installatie kwijt) en niets dat niet opgelost kan worden door het gewoon eens opnieuw te proberen, en nog eens, ... Het was zelfs zo'n groot succes dat we al aan het plannen waren voor een Linux Reinstall Party.

Reviewer: ```suggestion
Net als elk jaar duwen *ik bedoel* begeleiden we weer een hoop mensen in de wondere wereld van Linux. Dit jaar zonder al te grote problemen (lees: niet *iedereen* was zijn/haar installatie kwijt). Niets dat niet opgelost kan worden door het gewoon eens opnieuw te proberen, en nog eens, ... Het was zelfs zo'n groot succes dat we al aan het plannen waren voor een Linux Reinstall Party.
```

Author: niet akkoord, ik vind dwingen hier beter passen...

Reviewer: :+1:

### Geeks, Attack! VII (26 oktober)

> Nog een jaarlijkse traditie is de grote LAN-party in de Therminal. Wat is er leuker dan een nachtje door je favoriete videogames te spelen? Dit samen met andere mensen doen? Toernooien met heuse prijzen? Pizza's die voor jou geleverd worden? De LAN-party had dit allemaal!

Reviewer 2: ```suggestion
Een van onze andere jaarlijkse tradities is de grote LAN-party in de Therminal. Wat is er leuker dan een nachtje door je favoriete videogames te spelen? Dat samen met andere mensen doen? Deelnemen aan toernooien met heuse prijzen? Pizza's die voor jou geleverd worden? De LAN-party had dit allemaal!
```

### Zeus bezoekt het datacenter (5 november)

Wie al eens uit het raam van onze kelder gekeken heeft, heeft waarschijnlijk het bordje S10 al wel eens gezien. De legende gaat dat dit gebouw een bron is van data en dat de krachtigste computers uit de buurt daar bewaard worden! Tijd voor Zeus om daar eens te gaan kijken dus. Met de hulp van de werknemers van de HPC kregen we een rondleiding én de kans om honderden flikkerende lichtjes te zien. Wat wil een informaticus nog meer?

### OSM Worksops (14 en 28 november)

> Omdat open source/open data toch wel een belangrijk deel van Zeus is, organiseerden we samen met Pieter Vander Vennet een introductie tot de wereld van OpenStreetMap. In de eerste workshop leerden we mensen hoe ze zelf data toevoegen om zo de wereld een beetje verder in kaart te brengen. In de tweede leerden we werken met OsmAnd, de open routeplanner die draait op OpenStreetMap. We leerden hoe we OsmAnd kunnen 'hacken' om het zo te maken als we zelf willen.

Reviewer 2: ```suggestion
Omdat open source/open data toch wel een belangrijk deel van Zeus is, organiseerden we samen met Pieter Vander Vennet een introductie tot de wereld van OpenStreetMap. In de eerste workshop leerden we mensen hoe ze zelf data toevoegen om zo de wereld een beetje verder in kaart te brengen. In de tweede workshop leerden we werken met OsmAnd, de open routeplanner die draait op OpenStreetMap. We leerden hoe we OsmAnd kunnen 'hacken' om het zo te maken als we zelf willen.
```

### Pannenkoeken en Jenever Codenight

> **Ballmer-peak**: The theory that computer programmers obtain quasi-magical, superhuman coding ability when they have a blood alcohol concentration percentage between 0.129% and 0.138%.

> Ook deze keer sloten we het kalenderjaar af met een avond vol pannenkoeken, jenever en code. Dit jaar hebben we het ook gecombineerd met D-Ploy Day, wat inhield dat we zo veel mogelijk Zeusprojecten op zo veel mogelijk laptops lieten draaien om zo mensen te stimuleren om aan deze projecten te sleutelen. Zoals iedereen weet, Ruby wordt sneller gedeployed als men al een paar shotjes op heeft ... toch?

Reviewer 2: ```suggestion
Ook deze keer sloten we het kalenderjaar af met een avond vol pannenkoeken, jenever en code. Dit jaar hebben we het ook gecombineerd met D-Ploy Day, wat inhield dat we zo veel mogelijk Zeusprojecten op zo veel mogelijk laptops lieten draaien om zo mensen te stimuleren om aan deze projecten te sleutelen. Zoals iedereen weet, wordt Ruby sneller gedeployed als men al een paar shotjes op heeft ... toch?
```

> Ook deze keer sloten we het kalenderjaar af met een avond vol pannenkoeken, jenever en code. Dit jaar hebben we het ook gecombineerd met D-Ploy Day, wat inhield dat we zo veel mogelijk Zeusprojecten op zo veel mogelijk laptops lieten draaien om zo mensen te stimuleren om aan deze projecten te sleutelen. Zoals iedereen weet, Ruby wordt sneller gedeployed als men al een paar shotjes gedaan heeft, ,toch?

Reviewer: ```suggestion
Ook deze keer sloten we het kalenderjaar af met een avond vol pannenkoeken, jenever en code. Dit jaar hebben we het ook gecombineerd met D-Ploy Day, wat inhield dat we zo veel mogelijk Zeusprojecten op zo veel mogelijk laptops lieten draaien om zo mensen te stimuleren om aan deze projecten te sleutelen. Zoals iedereen weet, Ruby wordt sneller gedeployed als men al een paar shotjes op heeft ... toch?
```

### IHNGTWHSIPB-codenight (13 februari)

> Bij het zien van die naam denk je misschien "Wie heeft zijn toetsenbord laten vallen en dat als naam gekozen?", maar het is weldegelijk een afkorting. De "Ik Heb Nog Genoeg Tijd Want Het Semester Is Pas Begonnen"-codenight. Het tweede semester kwam op gang met een codenight gericht op iedereen die zich niet kon inhouden om te programmeren zonder dat er een deadline en punten boven het hoofd hing. Want wat doen Zeusers en Zeusinnen buiten de verplichte portie programmeren? Meer programmeren natuurlijk!

Reviewer 3: ```suggestion
Bij het zien van die naam denk je misschien "Wie heeft zijn toetsenbord laten vallen en dat als naam gekozen?", maar het is weldegelijk een afkorting. De "Ik Heb Nog Genoeg Tijd Want Het Semester Is Pas Begonnen"-codenight. Het tweede semester kwam op gang met een codenight gericht op iedereen die zich niet kon inhouden om te programmeren zonder dat er hen een deadline en punten boven het hoofd hingen. Want wat doen Zeusers en Zeusinnen buiten de verplichte portie programmeren? Meer programmeren natuurlijk!
```

### HTTPizza/2.0 (21 februari)

> Samen met IDLab organiseerden we dit jaar de opvolger van HTTPizza in 2017. Met coole challanges in het thema van een pizzeria lieten we onze leden kennis maken met de wondere wereld van het web door onze eigen HTTP-variant. Een PIZZA-request sturen is toch zo veel leuker dan een GET-request?

Reviewer 3: ```suggestion
Samen met IDLab organiseerden we dit jaar de opvolger van HTTPizza in 2017. Met coole challenges in het thema van een pizzeria lieten we onze leden kennis maken met de wondere wereld van het web door onze eigen HTTP-variant. Een PIZZA-request sturen is toch zo veel leuker dan een GET-request?
```

> Samen met IDLab organisserden we dit jaar de opvolger van HTTPizza in 2017. Met coole challanges in het thema van een pizzeria lieten we onze leden kennis maken met de wondere wereld van het web door onze eigen HTTP-variant. Een PIZZA-request sturen is toch zo veel leuker dan een GET-request?

Reviewer: ```suggestion
Samen met IDLab organiseerden we dit jaar de opvolger van HTTPizza uit 2017. Met coole challenges in het thema van een pizzeria lieten we onze leden kennis maken met de wondere wereld van het web door hen een eigen HTTP-variant te laten implementeren. Een PIZZA-request sturen is toch zo veel leuker dan een GET-request?
```

### Hashcode (28 februari)

> Ook dit jaar was Zeus van de partij op de Google Hashcode, de jaarlijkse challenge van Google waarbij teams uitgedaagd worden om een zo goed mogelijke oplossing te vinden voor een origineel probleem. We hadden dit jaar een team dat het tot de finale van de Hashcode gehaald heeft!

Reviewer: ```suggestion
Ook dit jaar was Zeus van de partij op de Google Hashcode, de jaarlijkse programmeerwedstrijd van Google waarbij teams uitgedaagd worden om een zo goed mogelijke oplossing te vinden voor een origineel probleem. We hadden dit jaar een team dat het tot de finale van de Hashcode geschopt heeft!
```

> Ook dit jaar was Zeus van de partij op de Google Hashcode, de jaarlijkse programmeerwedstrijd van Google waarbij teams uitgedaagd worden om een zo goed mogelijke oplossing te vinden voor een origineel probleem. We hadden dit jaar een team dat het tot de finale van de Hashcode gehaald heeft!

Reviewer 3: ```suggestion
Ook dit jaar was Zeus van de partij op de Google Hashcode, de jaarlijkse programmeerwedstrijd van Google waarbij teams uitgedaagd worden om een zo goed mogelijke oplossing te vinden voor een origineel probleem. We hadden dit jaar een team dat het tot de finale van de Hashcode geschopt heeft!
```

> Ook dit jaar was Zeus van de partij op de Google Hashcode, de jaarlijkse programmeerwedstrijd van Google waarbij teams uitgedaagd worden om een zo goed mogelijke oplossing te vinden voor een origineel probleem. We hadden dit jaar een team dat het tot de finale van de Hashcode geschopt heeft!

Reviewer 2: ```suggestion
Ook dit jaar was Zeus van de partij op de Google Hashcode, de jaarlijkse programmeerwedstrijd van Google waarbij teams uitgedaagd worden om een zo goed mogelijke oplossing te vinden voor een origineel probleem. We hadden dit jaar zelfs een team dat het tot de finale van de Hashcode geschopt heeft!
```

### Capture The Friday (7 maart)

Dit event viel zoals de naam al doet vermoeden op een vrijdag... Right? FOUT! Het was een donderdag! Hackers hebben vrijdag gestolen en het is aan ons om de dag te redden! Dit doen we door zoveel mogelijk dingen terug te hacken. Naast het mega-spannende verhaal was dit ook een super toffe capture the flag met challenges van verschillende moeilijkheden. Er werd ons ook nog eens duidelijk gemaakt waarom we geen PHP gebruiken.

> ### 12urenloop (29 april)

Reviewer: Misschien verwijzen naa de 12UL blogpost?

Author: Yes, maar die bestond nog niet toen ik deze blogpost geschreven had :D

Author: Ook, het was vooral over vorig jaar, dus denk ik dat een post over dit jaar niet helemaal relevant is...

Reviewer: Ok :)

> Ook dit jaar zat Zeus paraat op de Gentse 12urenloop om ons telsysteem bij te sturen waar nodig. Gelukkig hadden we dit jaar een goede backup aan onze manuele telling, want ons telsysteem lag zo'n beetje plat.

Reviewer 3: ```suggestion
Ook dit jaar zat Zeus paraat op de Gentse 12urenloop om ons telsysteem bij te sturen waar nodig. Gelukkig hadden we dit jaar een goede back-up aan onze manuele telling, want ons telsysteem lag zo'n beetje plat.
```

> Ook dit jaar zat Zeus paraat op de Gentse 12urenloop om ons telsysteem bij te sturen waar nodig. Gelukkig hadden we dit jaar een goede back-up aan onze manuele telling, want ons telsysteem lag zo'n beetje plat.

Reviewer 2: ```suggestion
Ook dit jaar was Zeus paraat op de Gentse 12urenloop om ons telsysteem bij te sturen waar nodig. Gelukkig hadden we dit jaar een goede back-up aan onze manuele telling, want ons telsysteem lag zo'n beetje plat.
```

> walkie: "ALLES IS DOWN, trek die kabel nu uit!"
>
> \*startschot klinkt\*

Gelukkig genoeg hebben we dankzij onze helpers die manueel geteld hebben en mensen erop gewezen hebben dat nummers aan de voorkant moeten hangen toch nog een correcte telling kunnen tonen!

## Projecten

Naast deze talrijke evenementen hebben we ook nog een paar projecten draaiend gehouden en zelf een paar nieuwe uit de grond gestampt. Tijd voor een lijstje!

### G2

> Wat begon als een idee van een paar eerstejaars op een septemberavond draaide dit jaar uit tot een echt beginnend project. We willen ervoor zorgen dat onze leden een "gateway" hebben tot de Zeusprojecten. G2 is hiervoor uitgedacht en opgesteld. Veel beter dan [deze blogpost](https://zeus.gent/blog/18-19/g2_blogpost/) ga ik het niet uitgelegd krijgen, dus kijk daar zeker eens naar!

Reviewer 2: ```suggestion
Wat begon als een idee van een paar eerstejaars op een septemberavond, draaide dit jaar uit tot een echt beginnend project. We willen ervoor zorgen dat onze leden een "gateway" hebben tot de Zeusprojecten. G2 is hiervoor uitgedacht en opgesteld. Veel beter dan [deze blogpost](https://zeus.gent/blog/18-19/g2_blogpost/) ga ik het niet uitgelegd krijgen, dus kijk daar zeker eens naar!
```

### Cat

> Katten! Wat hebben katten graag? Aandacht en affectie natuurlijk. Wij vonden dat zo'n dingen in kaart gebracht moeten worden, dus ontstond Cat. Je kan ermee knuffels aanvragen aan anderen en zien wie er met wie geknuffeld heeft. Waarom Cat? Wel, dat is een afkorting voor Cuddles And Bad Abbreviations natuurlijk.

Reviewer 2: ```suggestion
Katten! Wat hebben katten graag? Aandacht en affectie natuurlijk. Wij vonden dat zo'n dingen in kaart gebracht moeten worden, dus ontstond Cat. Je kan ermee knuffels aanvragen aan anderen en zien wie er met wie geknuffeld heeft. Waarom Cat? Wel, dat is een afkorting voor Cuddles And Bad Abbreviations, natuurlijk.
```

### Tappb

Tappb? Zijn er al niet genoeg ta[pb]+ varianten? Natuurlijk niet! Tappb is gemaakt door een paar tweedejaars informatici die voor het vak Software Engineering Lab een app gemaakt hebben voor Zeus die Tap en Tab combineert in een app. Hij is ook te vinden op de [Play Store](https://play.google.com/store/apps/details?id=gent.zeus.tappb)!

### MessageOS

> Omdat wij met Zeus enkel de meest state of the art technologie gebruiken. Hebben we ervoor gekozen om de messages die je vanop de Cammie-pagina kan sturen weer te geven via pure x86 assembly-code die op een floppy staat. Klinkt dat als een slecht idee? Wel, dat is het misschien ook, maar een leuk project was het zeker! We leerden veel bij over de mogelijkheden en (vooral) de limitaties van op zo'n laag niveau code te schrijven. Kom zeker eens kijken naar het scherm dat aan Hoek hangt, want je moet het met je eigen ogen zien om het te geloven.

Reviewer 2: ```suggestion
Omdat wij met Zeus enkel de meest state of the art technologie gebruiken, hebben we ervoor gekozen om de messages die je vanop de Cammie-pagina kan sturen weer te geven via pure x86 assembly-code die op een floppy staat. Klinkt dat als een slecht idee? Wel, dat is het misschien ook, maar een leuk project was het ook zeker! We leerden veel bij over de mogelijkheden en (vooral) de limitaties van op zo'n laag niveau code te schrijven. Kom zeker eens kijken naar het scherm dat aan Hoek hangt, want je moet het met je eigen ogen zien om het te geloven.
```
